src/site/static/apartment-details.liquid

```diff
@@ -104,7 +104,7 @@ eleventyComputed:
     {% assign occupancyLimit = units[0].occupancyLimit %}
     {% assign openStatus = units[0].openStatus %}
     {%- capture badgeMod -%}
-      {%- if openStatus == "Call for Status" -%}
+      {%- if openStatus == "Call for Availability" or openStatus == "Call for Status" -%}
```

Comment:
You should be able to also remove this "Call for Status" condition given line 123 in housing.js

Reply:
Fixed (https://github.com/theunitedeffort/theunitedeffort.org/pull/611/commits/5d37c68f79fe608ae30e617017fe2b51c42b6bb9)

Reply:
Fixed (https://github.com/theunitedeffort/theunitedeffort.org/pull/611/commits/5d37c68f79fe608ae30e617017fe2b51c42b6bb9)